docs/landing/data/releases.toml

```diff
@@ -1,5 +1,10 @@
 current = "3.12.0"
 
+[[versions]]
+  version = "4.0.0-beta"
```

Comment:
Might as well make it beta1, since that's what we'll call it.

Reply:
Done